src/ec/.storybook/preview-head.html

```diff
@@ -23,6 +23,7 @@
   sha512-am+swSO/CUqQMhuGjR4/njj5C7Blu2tpMMvk6NrnI/UJ1zjYE+R7BtaB7eAPZ7C1vr/s0SgJX0UeZ0lEyRe5+A=="
   crossorigin="anonymous"
 ></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.23.0/polyfill.min.js"></script>
```

Comment:
@papegaill, i also confirm this is working on EDGE and i've tried to load the polyfills in a different way (kinf of following this: https://github.com/storybookjs/storybook/issues/213) with no success. Clearly loading a script is sub-optimal, but we have to maintain/update the preview-head continuosly so i don't think is a big issue. As @kalinchernev told you there is a more recent version of the script and maybe we can just use that. ;)